azext_iot/tests/digitaltwins/test_dt_twin_lifecycle_int.py

```diff
@@ -42,7 +42,7 @@ def test_dt_twin(self):
             )
         )
         # Wait for RBAC to catch-up
-        sleep(20)
+        sleep(60)
```

Comment:
This didn't seem to help the `test_dt_twin_lifecycle_int` tests, but I'm not sure if that's because it actually needs more time, or if the timing of RBAC isn't the issue here. The failing command executes successfully locally after the test fails, so my assumption is the timing just needs to be tuned a bit more.

Reply:
Let's keep an eye on this, one potential cause I am thinking is a token caching issue. Looking into it a bit, interestingly enough, role-assignment propagation delays can take up to 30 minutes and for faster turn-around you can force a refresh by logging out and in again - [official guidance](https://docs.microsoft.com/en-us/azure/role-based-access-control/troubleshooting#role-assignment-changes-are-not-being-detected).